src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs

```diff
@@ -546,137 +543,197 @@ public enum WebRTCStatisticsProperty
         /// <summary>
         /// Timestamp.
         /// </summary>
-        Timestamp = WebRTCStatisticsPropertyCategory.Common | 0x01,
```

Comment:
changing the enum values are normally allowed, I will approve the PR only after confirming it is ok